clients/drcachesim/tests/scheduler_launcher.cpp

```diff
@@ -206,6 +214,12 @@ simulate_core(int ordinal, scheduler_t::stream_t *stream, const scheduler_t &sch
                 std::cerr << line.str();
             }
         }
+        if (type_is_instr(record.instr.type))
+            ++cur_segment_instrs;
+        if (cur_segment_instrs == op_print_every.get_value()) {
```

Comment:
nit: Maybe move this if-block inside the `type_is_instr` block too?

Reply:
Done.